3-get-session-info.md

```diff
@@ -0,0 +1,57 @@
+# Call Kobiton REST API to get session information
+This part will demonstrate how to get session information with Kobiton rest api. 
+>Kobiton have supported multiple language for [API documentation](add_api_doc_url). Go to docs for further language support
+
+## Prerequisites
+  - Kobiton installed.
```

Comment:
Install Kobiton desktop ??? As I know, call API doesn't need to install Kobiton desktop in local machine.